### PR TITLE
Fixes #200 Display notification in deployment pages

### LINKED ISSUE #200: Display notifications in deployment pages
### This is a
- [ ] Bug report
- [x] Feature request

### Description
At the current stage, the web application shows notifications of various results based on the different actions performed in the homepage. To maintain a constant flow of the application, we should also display notifications in the `deploy/github` and the `deploy/heroku` pages also.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #200 
## How Has This Been Tested?

Can be tested at https://yaydoc5.herokuapp.com/
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
       "required": true
     },
     "HOSTNAME": {
-      "description": "Hostname of your web application.",
+      "description": "Hostname of your web application."
```
Comment (imujjwal96) [inline app.json]: This was a bug in the app.json file that needed to be fixed for proper deployment of the app to Heroku

File: app.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ var passport = require("passport");
 var dotenv = require("dotenv");
 var session = require("express-session");
 
-require('./util/passport')(passport);
 dotenv.config({path: './.env'});
+require('./util/passport')(passport);
```
Comment (imujjwal96) [inline app.js]: had to shift the dotenv config line above passport instantiation as the latter could not receive the variables from `.env` file

File: views/deploy/heroku.jade [hunk 3]
```diff
@@ -16,13 +15,18 @@ block content
...
         });
 
-        data = {
-          'email': "#{email}",
-          'herokuAPIKey': "#{herokuAPIKey}",
-          'herokuAppName': herokuAppName,
-          'uniqueId': "#{uniqueId}"
-        };
-        socket.emit('heroku-deploy', data);
+        if (herokuAppName === "") {
```
Comment (pri22296) [inline views/deploy/heroku.jade]: This would be better suited to be in validation.js

Comment (imujjwal96) [inline views/deploy/heroku.jade]: @pri22296 Updated

**Comment by imujjwal96** [comment]
@sch00lb0y I hope you wouldn't mind reviewing this.

**Pull request merged**